### PR TITLE
story(EJ2-64131):Port the missing topics in filter & toolbar

### PR DESCRIPTION
**Pull request by SridharanJayabalan**
**Description**

Ported the missing topics in filter and toolbar topics in mvc & core platforms.

### CODE REVIEW

**Review by MahalakshmiKarthikeyan — APPROVED** [review]

**Comment by SridharanJayabalan** [comment]
rebuild

**Comment by SridharanJayabalan** [comment]
rebuild

**Comment by SridharanJayabalan** [comment]
rebuild

File: ej2-asp-core-mvc/gantt/filtering/filter-menu.md [hunk 1]
```diff
@@ -39,3 +39,38 @@ The Gantt control provides the menu filtering support for each column. You can e
 
 
 > The [`AllowFiltering`](https://help.syncfusion.com/cr/cref_files/aspnetcore-js2/Syncfusion.EJ2~Syncfusion.EJ2.Gantt.Gantt~AllowFiltering.html) property should be set to `true` to enable the filter menu. Setting the [`Columns.AllowFiltering`](https://help.syncfusion.com/cr/cref_files/aspnetcore-js2/Syncfusion.EJ2~Syncfusion.EJ2.Gantt.GanttColumn~AllowFiltering.html) property to `false` prevents rendering filter menu for a particular column.
+
+
+## Custom component in filter menu
+
+The [`column.filter.ui`](../api/gantt/column/#filter) is used to add custom filter components to a particular column.
+To implement custom filter ui, define the following functions:
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/filtering/filter-menu.md:47]: To implement a custom filter UI, define the following functions:

File: ej2-asp-core-mvc/gantt/filtering/filter-menu.md [hunk 2]
```diff
@@ -39,3 +39,38 @@ The Gantt control provides the menu filtering support for each column. You can e
...
+
+* `create`:  Creates custom component.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/filtering/filter-menu.md:49]: * `create`:  Creates a custom component.

File: ej2-asp-core-mvc/gantt/filtering/filter-menu.md [hunk 3]
```diff
@@ -39,3 +39,38 @@ The Gantt control provides the menu filtering support for each column. You can e
...
+* `write`: Wire events for custom component.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/filtering/filter-menu.md:50]: * `write`: Wires events for a custom component.

File: ej2-asp-core-mvc/gantt/filtering/filter-menu.md [hunk 4]
```diff
@@ -39,3 +39,38 @@ The Gantt control provides the menu filtering support for each column. You can e
...
+* `read`: Read the filter value from custom component.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/filtering/filter-menu.md:51]: * `read`: Reads the filter value from the custom component.

File: ej2-asp-core-mvc/gantt/filtering/filter-menu.md [hunk 5]
```diff
@@ -39,3 +39,38 @@ The Gantt control provides the menu filtering support for each column. You can e
...
+
+In the following sample, dropdown is used  as custom component in the TaskName column.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/filtering/filter-menu.md:53]: In the following sample, the dropdown is used as a custom component in the TaskName column.

File: ej2-asp-core-mvc/gantt/tool-bar.md [hunk 6]
```diff
@@ -1,6 +1,6 @@
 ---
 layout: post
-title: Tool Bar in ##Platform_Name## Gantt Component
+title: Toolbar in Syncfusion ##Platform_Name## Gantt Component
 description: Learn here all about Tool Bar in Syncfusion ##Platform_Name## Gantt component of Syncfusion Essential JS 2 and more.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/tool-bar.md:4]: description: Learn all about Tool Bar in Syncfusion ##Platform_Name## Gantt component of Syncfusion Essential JS 2 and more here.

File: ej2-asp-core-mvc/gantt/tool-bar.md [hunk 7]
```diff
@@ -9,7 +9,7 @@ documentation: ug
 ---
 
 
-# Toolbar
+# Toolbar in Gantt control
 
 The Gantt control provides the toolbar support to handle Gantt actions. The [`Toolbar`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Gantt.Gantt.html#Syncfusion_EJ2_Gantt_Gantt_Toolbar) property accepts the collection of built-in toolbar items and [`ItemModel`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Navigations.ToolbarItem.html) objects for custom toolbar items.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/tool-bar.md:14]: The Gantt control provides toolbar support to handle Gantt actions. The [`Toolbar`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Gantt.Gantt.html#Syncfusion_EJ2_Gantt_Gantt_Toolbar) property accepts the collection of built-in toolbar items and [`ItemModel`](https://help.syncfusion.com/cr/aspnetcore-js2/Syncfusion.EJ2.Navigations.ToolbarItem.html) objects for custom toolbar items.

File: ej2-asp-core-mvc/gantt/tool-bar.md [hunk 8]
```diff
@@ -153,3 +153,31 @@ You can enable or disable the toolbar items by using the `enableItems` method.
 {% endtabs %}
 {% endif %}
 
+
+
+## Add input elements to toolbar
+
+In the Gantt toolbar, you can add EJ2 editor elements like numeric text box, drop-down list, and date picker controls. The following code snippets demonstrates how to add EJ2 editors to the Gantt toolbar.
```
Comment (OnyimbiNelson) [inline ej2-asp-core-mvc/gantt/tool-bar.md:160]: In the Gantt toolbar, you can add EJ2 editor elements like a numeric text box, a drop-down list, and date picker controls. The following code snippets demonstrate how to add EJ2 editors to the Gantt toolbar:

**Comment by SridharanJayabalan** [comment]
rebuild

**Pull request merged**